src/features/defi/components/Approve/Approve.tsx

```diff
@@ -96,6 +102,31 @@ export const Approve = ({
             {translate('modals.approve.learnMore')}
           </Link>
         </Stack>
+        {!isUndefined(isExactAllowance) && (
```

Comment:
This can be a boolean or undefined according to typings, any reason we can use regular truthiness short-circuiting?
<img width="569" alt="image" src="https://user-images.githubusercontent.com/17035424/184397926-fc6a0737-3b9f-47d4-89aa-30e580ca8a72.png">

```suggestion
        {isExactAllowance && (
```

Reply:
@gomesalexandre what if `isExactAllowance` is false (the default value) but the toggle should be displayed? It won't display!

We need to verify that it's defined but not if it's false or true, this means that we want to display the switch toggle

Reply:
I see what this is doing now, thanks @NeOMakinG!
I'm wondering, should this prop ever be undefined, and if so, what's the rationale for it to be? You've mentioned that the default value is false, so we should never end up in an undefined prop state.
Is the reason why it is of type `isExactAllowance?: boolean`, that it's not implemented for all DeFi opportunities yet? If so, we should clearly comment why this check exist in this form, so we can eventually remove it, and not make the prop optional when implemented for all.

Reply:
FoxEthLP for example is using the same component but the exact allowance is not implemented

Reply:
I still added a comment

Reply:
LGTM now with the comment.
FYI, I'm not so concerned about this check being here (especially if it's temporary), but *why* it is here, and most importantly, for anyone else reading this to be able to get the full context without having to go through the full implementation across multiple modules before finding out why.